src/components/History/Timeline/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,55 @@
+import { Anchor } from '@mantine/core';
+import classes from './Navigation.module.css';
+
+export default function Navigation() {
+    const handleLinkClick = (year: string) => {
```

Comment:
Basically the first timeline item from each year is tagged with an `id` so we can scroll to it.